normalize-git-url.js

```diff
@@ -23,8 +23,10 @@ module.exports = function normalize (u) {
   if (altered) {
     if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
       returnedUrl = u.replace(/^git\+(.*:[^:]+):(.*)/, '$1/$2')
+    } else if (u.match(/^git\+file/)) {
+      returnedUrl = u.replace(/^git\+/, '')
     } else {
-      returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '')
+      returnedUrl = u.replace(/^(?:git\+)?ssh:[\/\\]{2}/, '')
```

Comment:
tbh, I'm a little itchy over this line, since it seems entirely unrelated to the other changes here? Maybe have this as a separate commit w/ its own test?